studio/src/app/models/data/slide.tsx

```diff
@@ -1,7 +1,8 @@
 export enum SlideTemplate {
     TITLE = 'title',
     CONTENT = 'content',
-    SPLIT = 'split',
+    SPLIT_HORIZONTALLY = 'split',
```

Comment:
Actually that was a bad idea from my side, sorry for that. We should stick to only `split` aka no refactoring.